src/sentry/event_manager.py

```diff
@@ -2047,8 +2052,8 @@ def _save_aggregate_performance(jobs: Sequence[Performance_Job], projects):
 
                     is_new = False
 
-                    description = performance_problems_by_fingerprint[existing_grouphash.hash].desc
-                    kwargs["data"]["metadata"]["title"] = f"N+1 Query: {description}"
+                    problem = performance_problems_by_fingerprint[existing_grouphash.hash]
+                    kwargs["data"]["metadata"] = get_metadata_for_performance_problem(problem)
```

Comment:
I don't think we can overwrite all the metadata like this, we still need the base metadata from the event

Reply:
Will let dan take over the review for this

Reply:
yeah I wasn't sure about it either 👍 
I've added a change where performance problem metadata is injected to the existing metadata